app/templates/people/person/positions/position/index.hbs

```diff
@@ -1,261 +1,154 @@
-                    <div class="au-o-grid__item au-u-1-4">
-                      <p><strong>Bestuursperiode</strong></p>
-                    </div>
-                    <div class="au-o-grid__item au-u-3-4">
-                      <p>{{@model.mandatory.mandate.governingBody.period}}</p>
-                    </div>
-                  </div>
-                </div>
-              {{/if}}
-              <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
-                <div class="au-o-grid">
-                  <div class="au-o-grid__item au-u-1-4">
-                    <p><strong>Geplande einddatum</strong></p>
-                  </div>
-                  <div class="au-o-grid__item au-u-3-4">
-                    <p>
-                      {{date-format @model.mandatory.expectedEndDate}}
-                    </p>
-                  </div>
-                </div>
-              </div>
-              {{#if @model.mandatory.typeHalf.label}}
-                <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
-                  <div class="au-o-grid">
-                    <div class="au-o-grid__item au-u-1-4">
-                      <p><strong>Verkozen</strong></p>
-                    </div>
-                    <div class="au-o-grid__item au-u-3-4">
-                      <p>
-                        {{@model.mandatory.typeHalf.label}}
-                      </p>
-                    </div>
-                  </div>
-                </div>
-              {{/if}}
-              {{#if @model.mandatory.endDate}}
-                <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
-                  <div class="au-o-grid">
-                    <div class="au-o-grid__item au-u-1-4">
-                      <p><strong>Einddatum</strong></p>
-                    </div>
-                    <div class="au-o-grid__item au-u-3-4">
-                      <p>
-                        {{date-format @model.mandatory.endDate}}
-                      </p>
-                    </div>
-                  </div>
-                </div>
-              {{/if}}
-              {{#if @model.mandatory.reasonStopped}}
-                <div class="au-o-grid__item au-u-1-1 au-u-1-2@medium">
-                  <div class="au-o-grid">
-                    <div class="au-o-grid__item au-u-1-4">
-                      <p><strong>Stopzetting</strong></p>
-                    </div>
-                    <div class="au-o-grid__item au-u-3-4">
-                      <p>
-                        {{@model.mandatory.reasonStopped}}
-                      </p>
-                    </div>
-                  </div>
-                </div>
-              {{/if}}
-            </div>
-          </div>
-        </c.content>
-      </AuCard>
-    </div>
+    <DataCard>
+      <:title>Aanstelling</:title>
+      <:card as |Grid|>
+        <Grid as |Item|>
+          <Item>
+            {{! TODO: Ember complains about having a component inside a label component }}
```

Comment:
What kind of component?

Reply:
This one `{{date-format @model.mandatory.startDate}}`